packages/test/mocha-test-setup/src/mochaHooks.ts

```diff
@@ -13,6 +13,7 @@ class TestLogger implements ITelemetryBufferedLogger {
         if (this.testName !== undefined) {
             event.testName = this.testName;
         }
+        event.hostName = "Mocha";
```

Comment:
can you set this to the pkgName, it exists in packageVersion.ts. do this for both